applications/system/js_app/modules/js_gui/js_gui.h

```diff
@@ -9,12 +9,15 @@ typedef enum {
     JsViewPropTypeString,
     JsViewPropTypeNumber,
     JsViewPropTypeArr,
+    JsViewPropTypeTypedArr,
+    JsViewPropTypeBool,
 } JsViewPropType;
 
 typedef union {
     const char* string;
     int32_t number;
     mjs_val_t array;
```

Comment:
More of a nitpick, but maybe it's a good idea to rename this field to `term` now that not just arrays could be stored here (typed arrays are not exactly arrays :smiling_face_with_tear:)
```suggestion
    mjs_val_t term;
```